content-src/components/DiscoveryStreamComponents/DSContextFooter/DSContextFooter.jsx

```diff
@@ -26,7 +26,7 @@ export class DSContextFooter extends React.PureComponent {
 
     return (
       <div className="story-footer">
-        {context && <p className="story-sponsored-label">{context}</p>}
+        {context && <p className="story-sponsored-label clamp">{context}</p>}
```

Comment:
clamp by itself here doesn't work, should set -webkit-line-clamp: 1 for story-sponsored-label